### DISCUSSION TITLE
Industry section is blocked.

### DISCUSSION THREAD

**Discussion by Chirag-197** [github/india#158] [Students]
I have successfully uploaded all the links my college Id card.But the Industry section is Blocked for me. I am not able to access the Industry section like how the other students are able to. what to do? @ey-mentor1, @ey-mentor2 , @EY-Mentor3, @EY-Mentor4.

**Comment by scmishra-cse** [comment]
Did you filled all the links and accounts I'd it was asking for??

> I have successfully uploaded all the links my college Id card.But the Industry section is Blocked for me. I am not able to access the Industry section like how the other students are able to. what to do? @ey-mentor1, @ey-mentor2 , @EY-Mentor3, @EY-Mentor4.

**Reply by Chirag-197** [reply]
@scmishra-cse  Yes i did upload all the links , I have uploaded them atleast 3 times till now and its still blocked for me
